src/hotspot/share/services/diagnosticCommand.hpp

```diff
@@ -431,7 +431,7 @@ class ThreadDumpDCmd : public DCmdWithParser {
   ThreadDumpDCmd(outputStream* output, bool heap);
   static const char* name() { return "Thread.print"; }
   static const char* description() {
-    return "Print all threads with stacktraces.";
+    return "Print all platform threads with stacktraces.";
```

Comment:
Probably best not hide this correction in this PR but file a separate issue for it.